spec/fixtures/responses/queries/freaks/freaks.json

```diff
@@ -64,86 +64,6 @@
                 "name": "Founder"
               },
               "level": {
-                "id": "3",
-                "name": "Advanced"
-              },
-              "norm": {
-                "id": "1",
-                "name": "full-time"
-              },
-              "specialties": [
-                {
-                  "id": "2",
-                  "name": "Backend"
-                }
-              ],
-              "technologies": [
-                {
-                  "id": "1",
-                  "name": "GraphQl"
-                },
-                {
-                  "id": "2",
-                  "name": "Ruby"
-                }
-              ]
```

Comment:
Why remove all of the stuff in this json file? They aren't related to the PR requirement.